core/domain/user_services_test.py

```diff
@@ -132,6 +135,53 @@ def test_get_user_id_from_username(self):
         self.assertIsNone(
             user_services.get_user_id_from_username('fakeUsername'))
 
+    def test_fetch_gravatar_success(self):
+        user_email = 'user@example.com'
+        self.enable_urlfetch_mock()
```

Comment:
Maybe you can set its content, status_code parameters here, too (in the with statement).